detekt-generator/src/main/kotlin/io/gitlab/arturbosch/detekt/generator/printer/defaultconfig/Exclusion.kt

```diff
@@ -44,15 +54,12 @@ private object TestExclusions : Exclusions() {
 }
 
 private object KotlinScriptExclusions : Exclusions() {
-
     override val pattern = "['**/*.kts']"
     override val rules = setOf("MissingPackageDeclaration")
 }
 
 private object KotlinScriptAndTestExclusions : Exclusions() {
-
-    override val pattern =
-        "['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**', " +
-            "'**/*.kts']"
+    override val pattern: String
+        get() = (testFolders + "**/*.kts").joinToString(prefix = "[", postfix = "]") { "'$it'" }
```

Comment:
I would do
```suggestion
        get() = (testFolders + "**/*.kts")
            .joinToString(prefix = "[", separator = ",", postfix = "]") { "'$it'" }
```
to remove implicitness, it's complex enough anyway

Reply:
Done